css/nav.css

```diff
@@ -0,0 +1,152 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+    /* --primary-color:#ff3f6c; */
+    --primary-color: rgb(9, 208, 243);
+    --secondary-color: #fff;
+    --text-color: #000;
+    --bg-secondary-color: #d8d8da;
+    --bg-error-color: rgb(253, 237, 237);
+    --error-txt-color: rgb(176 46 44);
+    --bg-warning-color: rgb(255, 244, 229);
+    --warning-txt-color: rgb(102, 60, 0);
+    --bg-info-color: rgb(229, 246, 253);
+    --info-txt-color: rgb(1, 67, 97);
+    --bg-success-color: rgb(237, 247, 237);
+    --success-txt-color:  rgb(30, 70, 32);
+  }
+
+html{
+    font-size: 62.5%;
+}
+
+body{
+    width: 100%;
+    min-height: 100vh;
+    height: max-content;
+    font-family: 'Nunito Sans', sans-serif;
+}
+
+.container{
+    min-height: 100%;
+    height: max-content;
+}
+
+.nav-bar{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+}
+
+.left-nav{
+    padding: 1rem;
+    margin-left: 1rem;
+    
+}
+
+.left-nav a{
+    font-size: 3rem;
+    text-decoration: none;
+    color: var(--text-color);
+    
+}
+.right-nav{
+     list-style: none;
+     display: flex;
+     flex-direction: row;
+     align-items: center;
+
+}
+
+.right-nav li{
+    margin-right: 2rem;
+    cursor: pointer;
+}
+
+.right-nav a{
+    text-decoration: none;
+    font-size: 1.6rem;
+    font-weight: 400;
+    color: var(--primary-color);
+}
+.right-nav li a:hover{
+   color: var(--text-color);
+}
+
+.toggle-btn{
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    display: none;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 35px;
```

Comment:
it would be better if you follow relative units convention throughout

Reply:
Yea, agree. Changed it. thanks :-)